### DISCUSSION TITLE
Feature Freeze towards the beta

### DISCUSSION THREAD

**Discussion by MitchellMarkGeorge** [MitchellMarkGeorge/TabButler#41] [Announcements]
Hey maintainers, 

We are officially 2 weeks out from the public beta launch 🎉 . With this in mind, I will be implementing a "feature freeze" so we can spend more time fixing bugs and fine-tuning the product that we have right now. The last feature that will be pushed will be #12 I am almost done with (just need to confirm the final UI. For the next 2 weeks, I will greatly appreciate it if you guys take the time and just use the extension as much as possible to uncover any bugs and issues that should be fixed before the launch. 

I am once again extremely grateful to all of you for taking the time to work on this product. We are truly making something great.

@mohitkyadav @andrei-g-git @VladBrok @dylanplayer

**Comment by mohitkyadav** [comment]
@MitchellMarkGeorge if you have not already planned, I can also recommend you to post a nice and shiny post on producthunt after launch.

**Reply by MitchellMarkGeorge** [reply]
Yea that is definitely something I am looking to do for bost the beta and 1.0 launch
